analysis/topEFT/datacard_maker.py

```diff
@@ -241,7 +253,7 @@ def export2d(h):
             else:
                 export1d(h_sm, 'data_obs', 'sm', fout)
 
-            if p in self.signal or self.rename[p] in self.signal:
+            if not self.do_sm and (p in self.signal or self.rename[p] in self.signal):
```

Comment:
could we define a nicely named variable for `(p in self.signal or self.rename[p] in self.signal)` to make this if statement easier to mentally parse?

Reply:
Done.